src/utils.js

```diff
@@ -42,12 +45,26 @@ export function useAnJurors() {
 export function usePostEmail() {
   const { account } = useWeb3Connect() || ''
   return useCallback(
-    async email =>
-      fetch(env('SUBSCRIPTIONS_URL'), {
-        method: 'post',
-        headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify({ email, address: account }),
-      }),
+    async email => {
+      let result
+      try {
+        const response = await fetch(env('SUBSCRIPTIONS_URL'), {
+          method: 'post',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify({ email, address: account }),
+        })
+        if (!response.ok) {
+          throw new Error(
+            `Received a wrong status when trying to subscribe: ${response.status}.`
+          )
+        }
+      } catch (err) {
+        Sentry.withScope(scope => {
+          scope.setUser({ email, username: account })
```

Comment:
We could also have used a custom field for `account`, but we won’t have a username so I think it’s ok to use the account for that.

Reply:
I agree!